hugolib/page.go

```diff
@@ -52,7 +52,7 @@ const (
 	// The rest are node types; home page, sections etc.
 	KindHome         = "home"
 	KindSection      = "section"
-	KindTaxonomy     = "taxonomy"
+	KindTaxonomy     = "taxtonomy"
```

Comment:
@vassudanagunta on another note: When I saw your name I thought of the Norwegian band https://www.vassendgutane.no/